src/test/java/wooteco/subway/admin/service/LineServiceTest.java

```diff
@@ -140,9 +141,28 @@ void removeLineStationAtTheEndOfLine() {
         assertThat(stationIds.get(1)).isEqualTo(2L);
     }
 
+    @Test
+    void findAllLinesWithStations() {
+        Line newLine = new Line(2L, "신분당선", "bg-green-500", LocalTime.of(05,30), LocalTime.of(22,30), 5);
```

Comment:
```suggestion
        Line newLine = new Line(2L, "신분당선", "bg-green-500", LocalTime.of(5,30), LocalTime.of(22,30), 5);
```

0으로 시작하는 숫자는 8진수를 나타냅니다.
09와 같은 값을 사용한다면 의도하지 않는 동작을 할 수 있겠네요.